README.md

```diff
@@ -35,7 +35,11 @@ Docker for development
 ----------------------
 
 0. Make sure you have [docker](https://docker.io) and [docker-compose](https://github.com/docker/compose)
-1. docker-compose up
+1. `docker-compose up`
+2. `docker exec -it snippets-service_web_1 /bin/bash`
```

Comment:
You can get a shell using docker-compose by running

`docker-compose run web bash`

Do you mind updating?

Reply:
Updated!

Reply:
ευχαριστώ!